### PR TITLE
NMS-9417 : use an internal radius server for radius tests

### PR DESCRIPTION
**Pull request by jm-k**
See https://issues.opennms.org/browse/NMS-9417
Added a maven dependency on tinyradius 1.0.1 (scope test)
Defined a mock radius server as Subclass of TinyRadiusServer
Used this internal radius server in tests (did not touch springsecurity however)

### CODE REVIEW

File: protocols/radius/pom.xml [hunk 1]
```diff
@@ -53,5 +53,18 @@
       <artifactId>org.opennms.core.test-api.services</artifactId>
       <scope>test</scope>
     </dependency>
+    <!-- https://mvnrepository.com/artifact/com.hynnet/tinyradius -->
```
Comment (j-white) [inline protocols/radius/pom.xml]: Can you fix the indentation to match the rest of the pom?

File: protocols/radius/src/test/java/org/opennms/protocols/radius/detector/RadiusAuthDetectorTest.java [hunk 2]
```diff
@@ -54,6 +56,7 @@
 
     @Autowired
     public RadiusAuthDetector m_detector;
+	private MockRadiusServer mockSrv = null;
```
Comment (j-white) [inline protocols/radius/src/test/java/org/opennms/protocols/radius/detector/RadiusAuthDetectorTest.java]: Indentation doesn't match the rest of the source file.

File: protocols/radius/src/test/java/org/opennms/protocols/radius/monitor/MockRadiusServer.java [hunk 3]
```diff
@@ -0,0 +1,39 @@
+package org.opennms.protocols.radius.monitor;
```
Comment (j-white) [inline protocols/radius/src/test/java/org/opennms/protocols/radius/monitor/MockRadiusServer.java]: Missing copyright header.

File: protocols/radius/pom.xml (hunk 1 above)
Comment (jm-k) [inline protocols/radius/pom.xml]: OK.

File: protocols/radius/src/test/java/org/opennms/protocols/radius/detector/RadiusAuthDetectorTest.java (hunk 2 above)
Comment (jm-k) [inline protocols/radius/src/test/java/org/opennms/protocols/radius/detector/RadiusAuthDetectorTest.java]: OK

File: protocols/radius/src/test/java/org/opennms/protocols/radius/monitor/MockRadiusServer.java (hunk 3 above)
Comment (jm-k) [inline protocols/radius/src/test/java/org/opennms/protocols/radius/monitor/MockRadiusServer.java]: OK.

Comment (jm-k) [inline protocols/radius/src/test/java/org/opennms/protocols/radius/monitor/MockRadiusServer.java]: OK. I did some modifications ....

Comment (jm-k) [inline protocols/radius/src/test/java/org/opennms/protocols/radius/monitor/MockRadiusServer.java]: P.R should be better now ...

**Comment by j-white** [comment]
Pushed to `pr/jira/NMS-9417`. Waiting on Bamboo.

**Pull request merged**